Tools/Scripts/webkitpy/api_tests/manager.py

```diff
@@ -151,9 +151,8 @@ def _print_tests_result_with_status(self, status, runner):
                 self._stream.writeln('')
 
     def _initialize_devices(self):
-        # FIXME: Remove the special-case for iOS Simulator once https://bugs.webkit.org/show_bug.cgi?id=256806 is resolved
         if self._port.port_name in ('iphone-simulator', 'ios-simulator'):
```

Comment:
If you take a look at `Tools/Scripts/webkitpy/port/ios_simulator.py`, you will notice that the `supported_device_types` for iOS Simulator already returns iPhone 12 as the default device. That means we can go a step further and just remove the whole `if` statement here, relying instead on the `if 'simulator' in self._port.port_name` condition.

Reply:
I think I have resolved your request with my recent change to this PR.